db_test.go

```diff
@@ -1778,12 +1778,6 @@ func Test_DB_Limiter(t *testing.T) {
 
 	for i := 0; i < 1024; i++ {
```

Comment:
Would choosing `k` random limits in this range rather than running the test for every single limit in range suffice?

Reply:
I fear that might bury a regression, and the test is cheap to run